monkey/monkey_island/cc/services/version_update.py

```diff
@@ -42,7 +43,11 @@ def _check_new_version():
         """
         url = VersionUpdateService.VERSION_SERVER_CHECK_NEW_URL % (env_singleton.env.get_deployment(), get_version())
 
-        reply = requests.get(url, timeout=15)
+        try:
+            reply = requests.get(url, timeout=7)
+        except requests.exceptions.RequestException:
+            logger.info("Can't get latest monkey version, probably no connection to the internet.")
+            raise NoInternetError
```

Comment:
This might result from lack of internet connection, but also if our update server fails to respond for some reason.
I would change the exception to something like `UpdateServerUnreachableException`

I think the term "update server" better describes the server than "version server", but it's more important to stay consistent. So either change the rest of the variable names here to "update server" or call the exception `VersionServerUnreachableException`